### PR TITLE
Create a main view and implement a theme

### PR DESCRIPTION
**Pull request by ch-iv**
Screen Shot 2024-11-26 at 9 24 10 PM

### CODE REVIEW

File: src/main/java/us/jonathans/view/CreateMatchView.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        gbc3.gridy = 1;
+        GridBagConstraints gbc4 = new GridBagConstraints();
+        gbc4.gridx = 3;
+        gbc4.gridy = 1;
+        JLabel phoneInputLabel = new JLabel("Phone");
+        JTextField phoneInputField = new JTextField(12);
+
+        add(phoneInputLabel, gbc3);
+        add(phoneInputField, gbc4);
+
```
Comment (jonathanqiao1) [inline src/main/java/us/jonathans/view/CreateMatchView.java:40]: remove whitespace

File: src/main/java/us/jonathans/view/CreateMatchView.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        setBorder(BorderFactory.createTitledBorder(viewName));
+        setLayout(new GridBagLayout());
+
+        GridBagConstraints gbc1 = new GridBagConstraints();
+        gbc1.gridx = 0;
+        gbc1.gridy = 0;
+        GridBagConstraints gbc2 = new GridBagConstraints();
+        gbc2.gridx = 1;
+        gbc2.gridy = 0;
+
```
Comment (jonathanqiao1) [inline src/main/java/us/jonathans/view/CreateMatchView.java:21]: remove whitespace

**Review by jonathanqiao1 — APPROVED** [review]
Beautiful view, however this conflicts with minas leaderboard panel, please discuss this further with mina verbally and through text

**Comment by ch-iv** [comment]
Thank you for your feedback. I implemented both changes!

**Pull request merged**